### ISSUE TITLE
podman-remote logs: --timestamps is a NOP

### ISSUE THREAD

**Issue by edsantiago** [containers/podman#15797]
Looks like nobody has actually ever tested `podman logs --timestamps`. 
```console
$ bin/podman-remote run --name foo  quay.io/libpod/testimage:20220615 date
Wed Sep 14 12:24:25 UTC 2022
$ bin/podman-remote logs --timestamps foo
Wed Sep 14 12:24:25 UTC 2022
```
Expected:
```
2022-09-14T06:24:25.569294000-06:00 Wed Sep 14 12:24:25 UTC 2022
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by Luap99** [comment]
Since you already have a test in your PR feels free to apply this fix so I don't have to create another PR for one line:
```diff
diff --git a/pkg/domain/infra/tunnel/containers.go b/pkg/domain/infra/tunnel/containers.go
index 324802a1f..572807ad6 100644
--- a/pkg/domain/infra/tunnel/containers.go
+++ b/pkg/domain/infra/tunnel/containers.go
@@ -517,7 +517,7 @@ func (ic *ContainerEngine) ContainerLogs(_ context.Context, nameOrIDs []string,
        stdout := opts.StdoutWriter != nil
        stderr := opts.StderrWriter != nil
        options := new(containers.LogOptions).WithFollow(opts.Follow).WithSince(since).WithUntil(until).WithStderr(stderr)
-       options.WithStdout(stdout).WithTail(tail)
+       options.WithStdout(stdout).WithTail(tail).WithTimestamps(opts.Timestamps)
 
        var err error
        stdoutCh := make(chan string)
```

**Comment by edsantiago** [comment]
Oh, nice, thank you! I will do so, but will wait until #15796 merges.

**Comment by edsantiago** [comment]
@Luap99 the timestamps I get now are 1-second resolution. With plain podman, I get sub-second:
```
remote: 2022-09-14T07:01:04          -06:00 hi    (whitespace added to highlight diff; not in original)
normal: 2022-09-14T07:01:04.964164000-06:00 hi
```
Is this intentional? Is it fixable?

**Comment by Luap99** [comment]
> Is this intentional? Is it fixable?

I have no idea. The timestamp is set on the server so it is part of the API, changing this format could be considered breaking?
Also maybe it is required for docker compat?

https://github.com/containers/podman/blob/83c148c4dd9c9ffb1b6a8f8436e86129067e3901/pkg/api/handlers/compat/containers_logs.go#L150
vs
https://github.com/containers/podman/blob/83c148c4dd9c9ffb1b6a8f8436e86129067e3901/libpod/logs/log.go#L205
https://github.com/containers/podman/blob/83c148c4dd9c9ffb1b6a8f8436e86129067e3901/libpod/logs/log.go#L22

**Comment by Luap99** [comment]
Looks like docker is using the local podman format:
https://github.com/moby/moby/blob/924edb948c2731df3b77697a8fcc85da3f6eef57/pkg/jsonmessage/jsonmessage.go#L17
https://github.com/moby/moby/blob/7b9275c0da707b030e62c96b679a976f31f929d3/api/server/httputils/write_log_stream.go#L51

I guess we could change the remote format to the local to better match docker?

**Comment by edsantiago** [comment]
Fixed in #15794 but the mergebot didn't seem to catch my edit.
